how-to-docs/kubernetes-token.md

```diff
@@ -0,0 +1,15 @@
+# How to get a kubernetes token for the UK cluster
+
+Log into the [platform hub][platform hub link].
+
+Select **Support Requests** under **Help & Support** from the sidebar.
+
+Select _Kubernetes token request_ from the list of available request templates.
+
+Select the required cluster from the drop down list and submit your request.
```

Comment:
shouldn't cluster be namespace?

Reply:
Yes, thank you

Reply:
The existing support request form is outdated (so it needs updating); the only thing a user needs to provide now is what cluster they would like access to. Actual role bindings to namespaces is done separately (as a PR to the relevant Gitlab repo).

In the future hopefully we'll combine these two steps into one flow in the hub. But for now they are separate.